go.mod

```diff
@@ -0,0 +1,45 @@
+module github.com/mendersoftware/workflows
+
+go 1.13
+
+require (
+	github.com/DataDog/zstd v1.4.5 // indirect
```

Comment:
why are there so many indirect modules here?

Reply:
that is just the result of running `go mod tidy` - unfortunately I don't really know how the go dependency resolution works, however this should decrease the number of dependencies we need to care about 🤷‍♂️

Reply:
ahh, now I see what's happening. `go mod` is recreating the `govendor` vendor/ tree. I suppose we can remove the old vendor tree before initializing `go mod` to remove these indirect imports (just be careful to pin the incompatible repositories to a commit hash).